libraries/nexus_helper.rb

```diff
@@ -37,5 +37,13 @@ def wait_until_ready!(endpoint, timeout = 15 * 60)
     rescue Timeout::Error
       raise Nexus3NotReady.new(endpoint, nexus_timeout)
     end
+
+    # Load Groovy scripts via the Nexus3 API.
+    class Scripting
```

Comment:
This class could be in its own file `libraries/scripts_helper.rb` and namespace `Nexus3::Scripts`
What do you think?

Reply:
Ok, done.